automation/devops_automation_infra/utils/container.py

```diff
@@ -25,3 +26,28 @@ def start_container_by_service(host, name):
         raise NotImplementedError("Start container not implemented yet")
     else:
         host.Docker.run_container_by_service(name)
+
+
+def is_docker(host):
+    try:
+        host.SshDirect.execute("which docker")
+    except SSHCalledProcessError:
+        return False
+    return True
+
+
+def is_crio(host):
+    try:
+        host.SshDirect.execute("which crictl")
+    except SSHCalledProcessError:
+        return False
+    return True
+
+
+def get_container_engine(host):
+    if is_crio(host):
+        return "crio"
```

Comment:
this means that a machine which has crio and docker as well will return crio.. FYI